x-pack/legacy/plugins/actions/server/action_type_registry.ts

```diff
@@ -61,6 +61,14 @@ export class ActionTypeRegistry {
       [`actions:${actionType.id}`]: {
         title: actionType.name,
         type: `actions:${actionType.id}`,
+        maxAttempts: actionType.maxAttempts || 1,
+        getRetry(attempts: number, error: any) {
+          if (error instanceof ExecutorError) {
+            return error.retry == null ? true : error.retry;
+          }
+          // Retry other kinds of errors
+          return true;
```

Comment:
I _think_ we want to have this return false by default.  Idea being that an action will know when something is retry-able - setting retry to `true` or a Date - and any other case, should not be retryable.

Reply:
Yeah I guess by looking here https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/actions/server/lib/execute.ts#L29-L74 any code that could throw here is already being handled and wrapped into a proper `ExecutionError`. Some may need to return `retry: false` ex validation?

Reply:
I went ahead and made it `false` for other kinds of errors. I can't think of a reason to retry totally unhandled errors.